src/components/page/index.js

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+import IconButton from '../iconButton';
+import classes from './styles.module.css';
+import DarkModeIcon from '../../assests/darkModeIcon';
+import LightModeIcon from '../../assests/lightModeIcon';
+import { themes, getTheme, setTheme } from '../../utils/theme';
+
+
+const Page = props => {
+    const [darkMode, setDarkMode] = useState(false);
+
+    useEffect(() => loadTheme(), []);
+
+    const loadTheme = () => {
+        const currTheme = getTheme();
+        setDarkMode(currTheme === themes.DARK ? true : false);
```

Comment:
currTheme === themes.DARK already evaluates to true or false, no need for explicitly setting true or false

Reply:
done